src/cirrus/build.py

```diff
@@ -233,5 +250,58 @@ def main():
         build_docs(make_opts=opts.docs)
 
 
+class Requirements:
+    """
+    Modify requirments files
+    """
+    def __init__(self, dep_filename, sub_dep_filename, activate_cmd):
```

Comment:
Add some docstrings for the arguments here.
Also, explain what "direct" means.

Reply:
17675d5
1a292fb